### PR TITLE
tdx/scripts

### PR DESCRIPTION
**Pull request by Le-Zheng**
## Description

ppml/tdx/scripts/registry-deployment.sh 
ppml/tdx/scripts/verdictd-deployment.sh
ppml/tdx/scripts/encrypt-image-publish.sh

### CODE REVIEW

**Comment by Le-Zheng** [comment]
image

File: ppml/tdx/scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                }
+        }
+}
+EOF
+```
+
+#### Generate a encryption key
+```bash
+cat <<- EOF >/opt/verdictd/keys/84688df7-2c0c-40fa-956b-29d8e74d16c0
+1234567890123456789012345678901
```
Comment (glorysdj) [inline ppml/tdx/scripts/README.md:50]: change plaintext key to "YOUR-KEY"?

Comment (glorysdj) [reply ppml/tdx/scripts/README.md:50]: why 84688df7-2c0c-40fa-956b-29d8e74d16c0, hard coded uuid?

Comment (Le-Zheng) [reply ppml/tdx/scripts/README.md]: Updated. Test new script:
image

**Pull request merged**